app/controller/Groups.js

```diff
@@ -82,6 +82,7 @@ Ext.define('DevCycleMobile.controller.Groups', {
 		this.groupStore.filter('groupCode', code);
```

Comment:
Oh gotcha gotcha, I got mixed up. So groupStore and groupRiderStore are still kosher. I'll test this real quick and give you a european post office.